playbooks/common/openshift-master/config.yml

```diff
@@ -38,6 +38,11 @@
           public_hostname: "{{ openshift_public_hostname | default(None) }}"
           public_ip: "{{ openshift_public_ip | default(None) }}"
           deployment_type: "{{ openshift_deployment_type }}"
+      - role: nfs
+        local_facts:
+          host: "{{ openshift_nfs_host | default(None) if openshift_nfs_host is defined else groups.oo_nfs_to_config.0 if groups.oo_nfs_to_config | length > 0 else None }}"
```

Comment:
The fact we should persist, should probably be something like this:
```
storage_providers:
- name: abc 
  type: nfs
  exports_dir: <some dir>
- name: def
  type: <blah>
  <some_attribute>: <some_value>
```
and maybe another fact for mapping storage providers to volumes to use:
```
storage_volumes:
- name: abc1
  provider_name: abc
- name: def1
  provider_name: def
```